rai/node/node.cpp

```diff
@@ -3601,13 +3601,13 @@ void rai::election::confirm_if_quorum (rai::transaction const & transaction_a)
 	{
 		sum += i.first;
 	}
-	if (sum >= node.config.online_weight_minimum.number () && !(*block_l == *status.winner))
+	if (sum >= node.config.online_weight_minimum.number () && block_l->hash () != status.winner->hash ())
```

Comment:
Is this already done as a part of PR 916 ?

https://github.com/nanocurrency/raiblocks/pull/916/files#diff-ced934b3fddcffa6f71a6c8fc44ff563L501

Either way it's more clearly written like this to me, but I want to make sure that if we need further changes we identify them.